src/WireMock.Net.xunit/TestOutputHelperWireMockLogger.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using Newtonsoft.Json;
+using Stef.Validation;
+using WireMock.Admin.Requests;
+using WireMock.Logging;
+using Xunit.Abstractions;
+
+namespace WireMock.Net.xunit;
```

Comment:
I think the namespace should be `WireMock.Net.Xunit` to follow the same naming as the xUnit project.